src/preload/moduleOverrides/custom-save.replacements

```diff
@@ -0,0 +1,77 @@
+# Custom Save Replacements
+
+*plugin* `custom-save`
+
+## Add a checkbox to the save dialog
+
+*module* `custom-save`
```

Comment:
Not sure if fireflame cares about this anymore, but I think you're overriding `calculator-shell/save-dialog`

Reply:
It's ok. The specific module names are pretty irrelevant now. They can probably be deleted throughout, but they're not doing much harm.